core-services/nvidia_gpu.tf

```diff
@@ -0,0 +1,80 @@
+resource "kubernetes_daemonset" "nvidia" {
+  count = try(var.node_groups.additional_node_groups.0.compute_type == "GPU" ? 1 : 0, 0)
+  metadata {
+    name      = "nvidia-device-plugin-daemonset"
+    namespace = kubernetes_namespace_v1.md-core-services.metadata.0.name
+    labels = {
```

Comment:
We should be adding all of our default tags as labels

Reply:
Added